### DISCUSSION TITLE
Need help with date format

### DISCUSSION THREAD

**Discussion by telometto** [tfeldmann/organize#133] [Q&A] [answered]
Hey, guys!

`{created.month}` and `{created.day}` create folders without the tens and I'd like to know if I can make `organize` include the tens when it creates folders?

I.e. if `organize` creates a folder using `{created.year}{created.month}{created.day}`, the folder name is '202151' but I want it like '20210501'.

Is this possible? Thanks!

**Comment by tfeldmann** [comment, marked as answer]
Have a look at the [last example here](https://organize.readthedocs.io/en/latest/filters/#created):

In the new version you can now use `{created.strftime('%Y-%m-%d')}`

**Reply by telometto** [reply]
Thanks 😊
